portality/static/widget/fixed_query.js

```diff
@@ -1,9 +1,11 @@
 (function() {
     // Load the widget's js scripts then HTML content. We assume jQuery is available.
     jQuery(document).ready(function($) {
-        $('#doaj-fixed-query-widget').load(doaj_url + "/static/widget/fixed_query_body.html");
+        $('body').load(doaj_url + "/static/widget/fixed_query_body.html");
```

Comment:
What's the reason for this change?  It worked before, and I'm a bit wary of injecting dom modifications outside the div that we've asked embedders to create.